nxos/acl.j2

```diff
@@ -1,7 +1,8 @@
+{% if config_context["acl"]["definitions"] is defined and config_context["acl"]["definitions"]["named"] is defined %}
```

Comment:
```suggestion
{% if config_context.get("acl", {}).get("definitions", {}).get("named", {}} %}
```

Reply:
We tried to keep it consistent with the existing codebase, see for example: https://github.com/nautobot/demo-gc-templates/blob/main/cisco_nxos.j2#L93-L95